libs/shared/utils-accessibility/.eslintrc

```diff
@@ -0,0 +1 @@
+{ "extends": "../../../.eslintrc", "rules": {}, "ignorePatterns": ["!**/*"] }
```

Comment:
This entire library is probably pointless.

Reply:
removed